src/Sylius/Bundle/ChannelBundle/Behat/ChannelContext.php

```diff
@@ -53,7 +53,7 @@ public function channelHasFollowingProductsAssigned($code, TableNode $table)
 
         foreach ($table->getHash() as $product) {
             /** @var ProductInterface $product */
-            $product = $this->getRepository('product')->findOneBy(array('name' => $product));
+            $product = $this->getRepository('product')->findOneByName($product['product']);
```

Comment:
Why using `__call()` magic instead of clear (existing) method call with parameters?